test/models/term_test.exs

```diff
@@ -21,13 +20,41 @@ defmodule CoursePlanner.TermTest do
     refute changeset.valid?
   end
 
+  test "changeset with valid holidays" do
```

Comment:
could you remove these two tests "changeset with valid holidays" and "changeset with invalid holidays"?

they shouldn't be here, this probably came from wrong merge